tests/normal.t.pl

```diff
@@ -28,10 +28,10 @@ sub runTests
     my $hardlinks = 1;
     if($mode eq 'standard')
     {
-        &mount("--standard");
+        &mount("--standard -o direct_io");
     } elsif($mode eq 'paranoia')
     {
-        &mount("--paranoia");
+        &mount("--paranoia -o direct_io");
```

Comment:
I don't think this is a good idea. The tests should run with the options that are actually used in the wild.